Makefile

```diff
@@ -141,8 +141,8 @@ core: $(CACHE)/.setup
 
 apps: $(CACHE)/.setup
 	@$(ECHO) $(PRINT_RECIPE)
-	@cd $(SRC_DIR)/playActivityUI && BUILD_DIR="$(PACKAGES_APP_DEST)/Activity Tracker/App/PlayActivity" make
-	@find $(SRC_DIR)/playActivityUI -depth -type d -name res -exec cp -r {}/. "$(PACKAGES_APP_DEST)/Activity Tracker/App/PlayActivity/res/" \;
+	@cd $(SRC_DIR)/activity_tracker && BUILD_DIR="$(PACKAGES_APP_DEST)/Activity Tracker/App/ActivityTracker" make
+	@find $(SRC_DIR)/activity_tracker -depth -type d -name res -exec cp -r {}/. "$(PACKAGES_APP_DEST)/Activity Tracker/App/ActivityTracker/res/" \;
```

Comment:
These folder names should not be changed. Package manager will not be able to update users' old apps.